src/REVLoans.sol

```diff
@@ -829,7 +829,7 @@ contract REVLoans is ERC721, ERC2771Context, IREVLoans, Ownable {
                         terminals: terminals,
                         prices: controller.PRICES(),
                         tokens: controller.TOKENS()
-                    }) < newBorrowAmount
+                    }) != newBorrowAmount
```

Comment:
Hmm I feel like this might be a tough check, since if a revnet has a few different assets then oracles are going to constantly change the amount that can be borrowed. 

Perhaps we should turn `newBorrowAmount` into `minBorrowAmount` and verify that `_borrowableAmountFrom(..)` is more, and then use the amount that can be borrowed.

Reply:
good thinking. ill make this change tomorrow